script/RumpelConfig.sol

```diff
@@ -40,6 +40,12 @@ library RumpelConfig {
     address public constant MAINNET_ZIRCUIT_RESTAKING_POOL = 0xF047ab4c75cebf0eB9ed34Ae2c186f3611aEAfa6;
     address public constant MAINNET_SYMBIOTIC_WSTETH_COLLATERAL = 0xC329400492c6ff2438472D4651Ad17389fCb843a;
     address public constant MAINNET_SYMBIOTIC_SUSDE_COLLATERAL = 0x19d0D8e6294B7a04a2733FE433444704B791939A;
+    address public constant MAINNET_SYMBIOTIC_WBETH_COLLATERAL = 0x422F5acCC812C396600010f224b320a743695f85;
+    address public constant MAINNET_SYMBIOTIC_SWELL_SWBTC_COLLATERAL = 0x9e405601B645d3484baeEcf17bBF7aD87680f6e8;
```

Comment:
The contract at this address doesn't appear to, in it's proxy, have transfer and approve functions. am i reading that right? https://etherscan.io/address/0x9e405601B645d3484baeEcf17bBF7aD87680f6e8#writeProxyContract

Reply:
good catch -- seems like the new vaults do not have tokenized shares. I'll review and likely remove transfer and approve on all of the newer "public - slashable" vaults.

Reply:
updated